traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/widgets/PurchaseFormWidget.java

```diff
@@ -143,25 +159,25 @@ public FormHeader getSectionHeader(int sectionId) {
     @Nullable
     @Override
     public FormMessage getMessage(int sectionId, int fieldId) {
-        if (currentError != null && currentError.getGroupId() == sectionId && currentError.getQuestionId() == fieldId) {
-            String message;
-
-            switch (currentError.getError()) {
-                case REGEX_MISMATCH:
-                    message = getContext().getString(R.string.regex_mismatch);
-                    break;
-                case REQUIRED:
+        if (lstCurrentErrors == null || lstCurrentErrors.isEmpty())
```

Comment:
It's better to wrap all sentences inside 'if' and 'else' into braces. Just to avoid stupid mistakes in future.